### PR TITLE
Further readme updates

### PR DESCRIPTION
**Pull request by davidmalloncares**
some further updates to the readmes for the Bolt and Puppet Device tutorials

### CODE REVIEW

File: docs/02-using-with-puppet-device/03-configure-puppet-device/README.md [hunk 1]
```diff
@@ -25,14 +25,13 @@ ssl_fingerprint: <certificate SHA256 fingerprint>
 
 The host is the IP address of the Palo Alto firewall you want to manage.
 
-By default, the module performs SSL verification. To disable this, put `ssl: false` in the remote section of the `inventory.yaml` file.
```
Comment (david22swan) [inline docs/02-using-with-puppet-device/03-configure-puppet-device/README.md]: Why was this line removed?

Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/03-configure-puppet-device/README.md]: inventory.yaml is not relevant in this workflow, I believe it is there as a result of a copy-paste from the Panos/Bolt tutorial

Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/03-configure-puppet-device/README.md]: Actually, reading a note further down that page it looks like that change should be done in /etc/puppetlabs/puppet/devices/firewall.example.com.conf if you want to disable, I'll update

Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/03-configure-puppet-device/README.md]: Reinstated that line but point it to the correct file

Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/03-configure-puppet-device/README.md]: @david22swan / @DavidS - does that look ok now?

Comment (DavidS) [reply docs/02-using-with-puppet-device/03-configure-puppet-device/README.md]: sweet

File: docs/02-using-with-puppet-device/01-install-prerequisites/README.md [hunk 2]
```diff
@@ -4,7 +4,11 @@ Before you begin, you need Ruby, Puppet and a Palo Alto firewall that you can te
 
 1. To check whether you have Ruby installed, run `ruby --version`. This command prints out the version of Ruby installed. If you do not have Ruby installed, see [Ruby installer](https://rubyinstaller.org/downloads/) for instructions.
```
Comment (DavidS) [inline docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: Same as the bolt tutorial: the puppet-agent packages ship a ruby, so no need to check for versions or install a separate ruby.

File: docs/02-using-with-puppet-device/01-install-prerequisites/README.md [hunk 3]
```diff
@@ -4,7 +4,11 @@ Before you begin, you need Ruby, Puppet and a Palo Alto firewall that you can te
...
 
-2. Install the latest Puppet agent. See [Installing agents](https://puppet.com/docs/puppet/6.4/install_agents.html) for instructions. You can check that you have installed the agent correctly by running `puppet --version`. This command prints out the Puppet version number installed.
+2. Install the latest Puppet agent. See [Installing agents](https://puppet.com/docs/puppet/6.4/install_agents.html) for instructions. You can check that you have installed the agent correctly by running `puppet --version`. This command prints out the Puppet version number installed. A usefull tip is to add the Puppet install directory to your path so Puppet commands can be called without having to specify the full path:
+
+```
+export PATH=/opt/puppetlabs/bin:$PATH
+```
 
 3. Download a Palo Alto VM. If you are a Palo Alto customer, you may have VMs that you can run in [Virtual Box](https://www.virtualbox.org/). Alternatively, you can get a free trial on the [AWS marketplace](https://aws.amazon.com/marketplace/seller-profile?id=0ed48363-5064-4d47-b41b-a53f7c937314). If you are a Puppet employee, we have licenses for VMs that you can run in Virtual Box, or you can get an image from vmpooler.
```
Comment (DavidS) [inline docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: Add "Of course you can also use a live device, if you have one." ?

File: docs/02-using-with-puppet-device/01-install-prerequisites/README.md [hunk 4]
```diff
@@ -2,17 +2,18 @@
...
 
 ```
 export PATH=/opt/puppetlabs/bin:$PATH
 ```
 
-3. Download a Palo Alto VM. If you are a Palo Alto customer, you may have VMs that you can run in [Virtual Box](https://www.virtualbox.org/). Alternatively, you can get a free trial on the [AWS marketplace](https://aws.amazon.com/marketplace/seller-profile?id=0ed48363-5064-4d47-b41b-a53f7c937314). If you are a Puppet employee, we have licenses for VMs that you can run in Virtual Box, or you can get an image from vmpooler. 
+2. Get a Palo Alto device to connect to. You can use a live device if you have one, otherwise you can:
+    - Download a Palo Alto VM. If you are a Palo Alto customer, you may have VMs that you can run in [Virtual Box]    (https://www.virtualbox.org/), or
+    - Get a free trial for a Palo Alto VM on the [AWS marketplace](https://aws.amazon.com/marketplace/seller-profile?id=0ed48363-5064-4d47-b41b-a53f7c937314), or 
+    - If you are a Puppet employee, we have licenses for VMs that you can run in Virtual Box, or you can get an image from      vmpooler.
```
Comment (Thomas-Franklin) [inline docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: Seems to have extra spaces between from and vmpooler

Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: @Thomas-Franklin that should be rectified now

File: docs/02-using-with-puppet-device/01-install-prerequisites/README.md (hunk 3 above)
Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: @DavidS - that should be better now

Comment (DavidS) [reply docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: The first virtualbox link is broken and I can't seem to find the option to use your physical device.

Comment (davidmalloncares) [reply docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: Good spot, I fixed the virtualbox link. In step #2 I have just said "2. Get a Palo Alto device to connect to. You can use a live device if you have one..." which is basically all they need to do at that step if they already have a device, if not then there are details on options for getting a VM

Comment (DavidS) [reply docs/02-using-with-puppet-device/01-install-prerequisites/README.md]: d'oh. missed that.
